src/reistijden_v1/views.py

```diff
@@ -13,6 +15,20 @@
 logger = logging.getLogger(__name__)
 
 
+def store_error_content(e, request):
+    """
+    In order to keep the content of the messages that resulted in errors, we store them in the container for now.
+    This is a massive hack and should be removed very soon. If you find this code and
+    """
+    error_type = e.__repr__().replace("'", '')
+    folder = '/tmp/errors/'
+    file_path = f'{folder}{datetime.now().isoformat()}-{error_type}.xml'
+    if not os.path.exists(folder):
+        os.makedirs(folder)
+    with open(file_path, 'w') as f:
+        f.write(request.body.decode("utf-8"))
```

Comment:
Deze hele functie wil je het liefst nog even in één grote try/catch gooien, zodat het in ieder geval geen invloed heeft op de response.